src/coreComponents/physicsSolvers/fluidFlow/ThermalCompositionalMultiphaseFVMKernels.hpp

```diff
@@ -280,9 +280,11 @@ class FaceBasedAssemblyKernel : public isothermalCompositionalMultiphaseFVMKerne
       : Base::StackVariables( size, numElems ),
       dCompFlux_dT( size, numComp ),
       energyFlux( 0.0 ),
-      dEnergyFlux_dP( size ),
-      dEnergyFlux_dT( size ),
-      dEnergyFlux_dC( size, numComp )
+      dEnergyFlux_dP( 2 ),
+      dEnergyFlux_dT( 2 ),
+      dEnergyFlux_dC( 2, numComp ),
```

Comment:
```suggestion
      dConvectiveEnergyFlux_dP( 2 ),
      dConvectiveEnergyFlux_dT( 2 ),
      dConvectiveEnergyFlux_dC( 2, numComp ),
```
Maybe we can rename the `energyFlux` as `convectiveEnergyFlux` to make the distinction with the new `conductiveEnergyFlux`.

Reply:
ah, yes, done.